maintenance-mode.php

```diff
@@ -44,3 +56,20 @@ function vip_maintenance_mode_load_plugin_textdomain() {
 	load_plugin_textdomain( 'maintenance-mode', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
 }
 add_action( 'init', 'vip_maintenance_mode_load_plugin_textdomain' );
+
+/**
+ * Displays a warning in the admin to inform users about the VIP_MAINTENANCE_MODE constant
+ *
+ * Only users who can activate plugins will see the warning.
+ *
+ * @since 0.1.1
+ */
+function vip_maintenance_mode_admin_notice__constant_not_set() {
+	if ( !current_user_can( 'activate_plugins' ) ) {
+		return;
+	}
+
+	$class = 'notice notice-warning';
+	$message = __( 'Maintenance Mode won\'t work until you set the VIP_MAINTENANCE_MODE constant to <code>true</code>.', 'maintenance-mode' );
+	printf( '<div class="%1$s"><p>%2$s</p></div>', $class, wp_kses( $message, array( 'code' => array() ), array() ) );
```

Comment:
Ah, well the Code Reference is strictly correct. The Codex is stepping ahead, because if you provide a blank array, `wp_kses()` uses the return value of `wp_allowed_protocols()` so it uses the return of that function for the documentation on the 3rd parameter. Strictly incorrect, but helpful to the developer to know that's what the default value will be if you don't provide one.